### PR TITLE
Add invoxia integration

### PR DESCRIPTION
**Pull request by ezlo-picori**
## Proposed change
I developed a new integration `invoxia` which integrates Invoxia™ GPS Tracker into Home-Assistant.
Here I add a short documentation page for this integration.
This replaces previous PR (#21098 ) as a domain change has been requested.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/63671
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3084
- This PR fixes or closes issue: #21098 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/invoxia.markdown [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: "Invoxia (unofficial)"
```
Comment (frenck) [inline source/_integrations/invoxia.markdown]:
```suggestion
title: "Invoxia"
```

File: source/_integrations/invoxia.markdown [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ha_quality_scale: platinum
+ha_config_flow: true
+ha_codeowners:
+  - '@ezlo-picori'
+ha_domain: invoxia
+ha_platforms:
+  - device_tracker
+---
+
+`invoxia` is an integration which offers tracking support using your GPS trackers from [Invoxia™](https://www.invoxia.com) into Home Assistant.
```
Comment (frenck) [inline source/_integrations/invoxia.markdown]:
```suggestion
Invoxia is an integration that offers tracking support using your GPS trackers from [Invoxia™](https://www.invoxia.com) into Home Assistant.
```

File: source/_integrations/invoxia.markdown [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+## Authentication
+
+During configuration, you simply need to use the username and password associated to the Invoxia™ account linked to your GPS trackers.
+All trackers associated to your account will be directly registered and accessible.
+
```
Comment (frenck) [inline source/_integrations/invoxia.markdown]: This is already implied
```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/invoxia.markdown (hunk 1 above)
Comment (ezlo-picori) [reply source/_integrations/invoxia.markdown:2]: Remove "unofficial" from the documentation title for invoxia integration

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**